packages/patternfly-4/react-table/src/components/Table/Table.d.ts

```diff
@@ -35,10 +35,16 @@ export interface ICell {
   props: Object;
 }
 
+export interface IRowCell {
+  title: ReactNode;
+  props: Object;
+}
+
 export interface IRow {
-  cells: Array<String>;
+  cells: Array<ReactNode | IRowCell>;
   isOpen: Boolean;
   parent: Number;
+  props: Object;
```

Comment:
this should be `object` capital `Object` is advised against. You might try `unknown` though. It is new and is better for these types of things.

Reply:
I was kinda thinking about `any` but at the same time trying to avoid it. `unknown` looks much better!